api/host.go

```diff
@@ -91,16 +88,7 @@ type (
 func DefaultSearchHostOptions() SearchHostOptions {
 	return SearchHostOptions{
 		Limit:      -1,
-		FilterMode: HostFilterModeAll,
-	}
-}
-
-func (opts GetHostsOptions) Apply(values url.Values) {
-	if opts.Offset != 0 {
-		values.Set("offset", fmt.Sprint(opts.Offset))
-	}
-	if opts.Limit != 0 {
-		values.Set("limit", fmt.Sprint(opts.Limit))
+		FilterMode: HostFilterModeAllowed,
```

Comment:
Can we do this? I think we can right but I wasn't 100% on that. We change the client too so I figured this would be ok.

Reply:
I'd prefer dropping `DefaultSearchHostOptions` altogether since we don't use it and keep the `Hosts()` calls we already have in testing. There is no harm in using the deprecated route until we completely get rid of it. This will make for a smaller diff.